test/checkpoint/test_checkpoint.c

```diff
@@ -420,6 +420,9 @@ wt_shutdown(void)
     if (g.conn == NULL)
         return (0);
 
+    if (g.opts.tiered_storage)
+        testutil_tiered_end(&g.opts);
+
```

Comment:
Looking at the code, there are three other places that should also get this change:
random_directio
schema_abort
timestamp_abort 
I'm surprised that we didn't hit it in timestamp_abort since that supports multiple iterations.

Reply:
All three of those other tests call `testutil_tiered_begin` in child processes that get killed rather than existing. So there isn't a place to do cleanup.

Reply:
Ok, then I am good here.